docker-compose.dev.yaml

```diff
@@ -149,6 +149,103 @@ services:
     command: >
       sh -c /drs/bento_drs/chord_drs/startup.sh
 
+  service-registry:
+    build:
+      context: $PWD/lib/service-registry
+      dockerfile: Dockerfile.dev
+      args:
+        BASE_IMAGE: ${BENTOV2_SERVICE_REGISTRY_BASE_IMAGE}
+        BASE_IMAGE_VERSION: ${BENTOV2_SERVICE_REGISTRY_BASE_IMAGE_VERSION}
+        REPO: ${BENTO_SERVICE_REGISTRY_REPO}
+        BRANCH: ${BENTO_SERVICE_REGISTRY_BRANCH}
+        TAG: ${BENTO_SERVICE_REGISTRY_TAG}
+    image: ${BENTOV2_SERVICE_REGISTRY_IMAGE}:${BENTOV2_SERVICE_REGISTRY_VERSION}
+    container_name: ${BENTOV2_SERVICE_REGISTRY_CONTAINER_NAME}
```

Comment:
Use the `extends` property instead, to not have to repeat all the configuration from the regular container